src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetVOD/index.spec.tsx

```diff
@@ -0,0 +1,172 @@
+        live_state: value,
+        recording_time: 10,
+      });
+
+      render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+      screen.getByText(/There is nothing to harvest/);
+
+      expect(
+        screen.queryByText('Harvesting in progress...'),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Create vod' }),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Publish vod' }),
+      ).not.toBeInTheDocument();
+
+      cleanup();
+    });
+  });
+
+  it('shows message to record the live if nothing is recorded', () => {
+    const values = [liveState.STOPPED, liveState.HARVESTED];
+    values.forEach((value) => {
+      const video = videoMockFactory({
+        live_state: value,
+        recording_time: undefined,
+      });
+
+      render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+      screen.getByText(/There is nothing to harvest/);
+
+      expect(
+        screen.queryByText('Harvesting in progress...'),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Create vod' }),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Publish vod' }),
+      ).not.toBeInTheDocument();
+
+      cleanup();
+    });
+  });
+
+  it('shows message to record the live if recorded time is less or equal 0', () => {
+    const values = [liveState.STOPPED, liveState.HARVESTED];
+    values.forEach((value) => {
+      const video = videoMockFactory({
+        live_state: value,
+        recording_time: 0,
+      });
+
+      render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+      screen.getByText(/There is nothing to harvest/);
+
+      expect(
+        screen.queryByText('Harvesting in progress...'),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Create vod' }),
+      ).not.toBeInTheDocument();
+      expect(
+        screen.queryByRole('button', { name: 'Publish vod' }),
+      ).not.toBeInTheDocument();
+
+      cleanup();
+    });
+  });
+
+  it('shows havest button if live is stopped and recorded time is greater than 0', () => {
+    const video = videoMockFactory({
+      live_state: liveState.STOPPED,
+      recording_time: 10,
+    });
+
+    render(wrapInIntlProvider(<DashboardVideoLiveWidgetVOD video={video} />));
+
+    screen.getByRole('button', { name: 'Create vod' });
+
+    expect(
+      screen.queryByText('Harvesting in progress...'),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByText(/There is nothing to harvest/),
+    ).not.toBeInTheDocument();
+    expect(
+      screen.queryByRole('button', { name: 'Publish vod' }),
+    ).not.toBeInTheDocument();
+  });
+
+  it('shows publish and download vod buttons when live is harvested and recorded time is greater than 0', () => {
```

Comment:
```suggestion
  it('shows publish and download vod buttons when live is harvested and recorded time is greater than 0', () => {
```
In the name of the test, you mention two buttons, did you omit to test one? I only see Publish vod one.